scripts/render.js

```diff
@@ -250,7 +250,7 @@ async function compile(page) {
 
             affiliate_signup_url  : `https://login.binary.com/signup.php?lang=${affiliate_language_code}`,
             affiliate_password_url: `https://login.binary.com/password-reset.php?lang=${affiliate_language_code}`,
-            affiliate_email       : 'partners@binary.com',
+            affiliate_email       : 'paymentagents@binary.com',
```

Comment:
I think we need to add a new key for `payment_agent_email`. The affiliate email is `partners@binary.com` and we need it in other pages

Reply:
I see. Let me add a new key